src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,55 @@
 package core.basesyntax;
 
-public class WorkWithFile {
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
 
+public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        String data = readDataFromFile(fromFileName);
+        writeDataToFile(toFileName, data);
+    }
 
+    private String readDataFromFile(String fromFileName) {
```

Comment:
you should split this method into two small ones, one for reading data from files and the second one for forming of a report based on read data